src/Microsoft.Health.Fhir.Core/Features/Validation/ResourceNotValidException.cs

```diff
@@ -41,7 +41,8 @@ public ResourceNotValidException(IEnumerable<ValidationFailure> validationFailur
                     Issues.Add(new OperationOutcomeIssue(
                             OperationOutcomeConstants.IssueSeverity.Error,
                             OperationOutcomeConstants.IssueType.Invalid,
-                            failure.ErrorMessage));
+                            failure.ErrorMessage,
+                            new[] { failure.PropertyName }));
```

Comment:
Lets only create the array if PropertyName is NotNullOrEmpty

Reply:
Good idea, fixed in 2b34f6d.